### PR TITLE
[docs] Add status header for datadog exporter

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** <Describe what has changed.>
Add status header for datadog exporter.

### CODE REVIEW

File: exporter/datadogexporter/README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Datadog Exporter
 
+| Status                   |                |
+| ------------------------ |----------------|
+| Stability                | [beta]         |
+| Supported pipeline types | trace, metrics |
```
Comment (TylerHelmuth) [inline exporter/datadogexporter/README.md:6]:
```suggestion
| Supported pipeline types | traces, metrics |
```

**Review by TylerHelmuth — APPROVED** [review]
/cc @KSerrania @mx-psi @gbbr @knusbaum as code owners to verify beta stability

File: exporter/datadogexporter/README.md (hunk 1 above)
Comment (mx-psi) [reply exporter/datadogexporter/README.md:6]: Thanks for the oping Tyler, we are discussing whether the alpha or beta definition is more appropriate here for now :)

**Review by mx-psi — APPROVED** [review]
There are some deprecated configuration settings that we will eventually remove but otherwise the exporter has been used in production settings for a while so beta is appropriate

**Review by codeboten — APPROVED** [review]

**Pull request merged**